EIPS/2020 Block Reward Reduction.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: <to be assigned>
+title: 2020 Block Reward Reduction to 0.5 ETH
+author: John Lilic @jlilic twitter.com/johnlilic johnlilicEIP@gmail.com, Jerome de Tychey @jdetychey twitter.com/jdetychey,  Others
+discussions-to: https://ethereum-magicians.org/t/eip-2878-block-reward-reduction-to-0-5-eth/
+status: Draft
+type: Standards Track
+category Core
+created: 2020-08-11
+---
+
+
+## Simple Summary
+
+This EIP will reduce the block reward paid to proof-of-work validators.  Reducing the block reward will maintain the status quo of periodic community-activated block reward reductions.  
+
+## Abstract
+Of the top 4 Proof-of-Work blockchains, Ethereum pays the highest inflation rate for block validation.  Blockchains both larger, and smaller than Ethereum pay lower rates without any adverse effects.  Over-paying for block validators is an inefficiency that negatively affects all ETH holders, as it unnecessarily inflates the montary base and reduces the purchasing power of all Ether holders.   This EIP will reduce the block reward to bring inflation in-line with Bitcoin, which is the largest cryptocurrency by market cap.   Block rewards will thus be adjusted to a base of 0.5ETH, uncle and nephew rewards will be adjusted accordingly.    
```

Comment:
I think block validation is wrong term here, since nodes are not paid for validating.

Reply:
We are not referencing or making any suggestion that nodes are paid for validating.  Nodes *verify* blocks, but mining nodes *validate* blocks. Can you elaborate further as to what language you would like to see here, and why?  Block validation is done by miners, or, more specifically, pool operators running a full node, or non-pool small miners who run a full node.  We make reference to "block validation" as it encompasses all various actors who may be validating through pooled or non-pooled mining.

Reply:
Verify and validate are isomorphic. I believe the correct terminology here is `s/block validators/miners`. "Mining" has been colloquially used to refer to all duties of a miner (including block validation). Block validation, on its own, is really just a subset of a miner's duty. If the miner isn't actively searching PoW solutions (e.g. simply acting as a "block validator"), it will never be paid and therefore can never be overpaid.